etc/profile-a-l/koi.profile

```diff
@@ -0,0 +1,58 @@
+# Firejail profile for koi
+# Description: Theme scheduling for the KDE Plasma Desktop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include koi.local
+# Persistent global definitions
+include globals.local
+
+noexec ${HOME}
+noexec /tmp
+
+# Restriction below breaks the program on Arch.
```

Comment:
Du you happen to know what in disable-common.inc is breaking it?

Reply:
I don't know.